learning-progression.md

```diff
@@ -201,11 +223,9 @@ function displayIcons(isHeart : boolean) {
     }
 }
    
```

Comment:
_The first line is called a **signature**. It contains ..._
_Arguments are **local variables**, meaning that the function is going to use them in its block of code. Like any other variable, arguments have types._
_Under the signature is the function's code block. Like all code blocks, it is composed of statements._
Reference the [page](https://makecode.microbit.org/javascript/functions).
Now explain "reuse" of functions by explaining "calls" and their syntax. Then reference the [page](https://makecode.microbit.org/javascript/call).